packages/loot-core/src/platform/server/fs/shared.ts

```
@@ -1,19 +1,21 @@
 // @ts-strict-ignore
+import { t } from 'i18next';
+
 import { join } from './path-join';
 
 let documentDir;
 export const _setDocumentDir = dir => (documentDir = dir);
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)